schematics/universal/src/files/prerender.ts

```diff
@@ -1,5 +1,5 @@
 const domino = require('domino');
-const fs = require('fs');
+const fs = require('fs-extra');
```

Comment:
This would require installing `fs-extra` as part of the schematic, right?

Reply:
Oh, I did this long time ago, I don't remmember if I actually did that. But I would say yes.

Reply:
I was thinking on closing this PR and apply the changes on a next one which addresses other bugfixes on the prerendering script.

This little change wouldn't be need since the prerendering script is gonna change after all. The import checker is really needed and therefore, thanks for pointing out.